src/Uno.UI/UI/Xaml/UIElement.Layout.netstd.cs

```diff
@@ -64,13 +64,13 @@ private void InvalidateMeasureDirtyPath()
 		[MethodImpl(MethodImplOptions.AggressiveInlining)]
 		internal void InvalidateParentMeasureDirtyPath()
 		{
-			if (this.GetParent() is UIElement parent)
+			if (this.GetParent() is UIElement parent) //TODO:MZ: Should this use VisualTree.GetParent as fallback?
```

Comment:
In which case should it be a fallback?

Reply:
Specifically this one - https://github.com/unoplatform/uno/pull/8185 . For items the `Parent` is not set, but `VisualTreeHelper.GetParent` can still reach the `ItemsControl`

Reply:
@jeromelaban ping 🙂

Reply:
Added to the epic #8978